app/profile/page.tsx

```diff
@@ -1,69 +1,173 @@
-      <div className="w-3xl max-w-4xl mx-auto space-y-6">
-        <UserProfileCard />
-        <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
-          <div className="bg-card p-4 rounded-lg shadow-sm border">
-            <div className="flex items-center justify-between">
-              <h3 className="text-sm font-medium">Total Tests</h3>
-              <Trophy className="h-4 w-4 text-muted-foreground" />
-            </div>
-            <div className="text-2xl font-bold mt-2">127</div>
-            <p className="text-xs text-muted-foreground">+12 from last week</p>
+    <div className="bg-[#1a1f2e] p-6 rounded-2xl shadow-lg border border-[rgba(0,217,255,0.1)] stat-card-glow relative overflow-hidden group hover:border-[rgba(0,217,255,0.3)] transition-all duration-300">
+      <div className="absolute inset-0 bg-gradient-to-br from-[#00d9ff]/5 via-transparent to-[#9d4edd]/5 opacity-0 group-hover:opacity-100 transition-opacity duration-300"></div>
+      <div className="relative z-10">
+        <div className="flex items-center justify-between mb-4">
+          <h3 className="text-sm font-semibold text-[#94a3b8] uppercase tracking-wide font-mono">{title}</h3>
+          <div className="p-2 bg-[#0f1419] rounded-lg border border-[rgba(0,217,255,0.2)]" style={{ color }}>
+            {icon}
           </div>
+        </div>
+        <div className="text-3xl font-bold font-mono text-white mb-2" style={{ color }}>
+          {value}
+        </div>
+        {subtitle && (
+          <p className="text-xs text-[#94a3b8]">{subtitle}</p>
+        )}
+      </div>
+    </div>
+  );
+};
 
-          <div className="bg-card p-4 rounded-lg shadow-sm border">
-            <div className="flex items-center justify-between">
-              <h3 className="text-sm font-medium">Average WPM</h3>
-              <Target className="h-4 w-4 text-muted-foreground" />
-            </div>
-            <div className="text-2xl font-bold mt-2">68</div>
-            <p className="text-xs text-muted-foreground">+5 from last week</p>
-          </div>
+interface ActivityItemProps {
+  test: MockTestResult;
+}
 
-          <div className="bg-card p-4 rounded-lg shadow-sm border">
-            <div className="flex items-center justify-between">
-              <h3 className="text-sm font-medium">Best Time</h3>
-              <Clock className="h-4 w-4 text-muted-foreground" />
-            </div>
-            <div className="text-2xl font-bold mt-2">2:34</div>
-            <p className="text-xs text-muted-foreground">Fastest completion</p>
-          </div>
+const ActivityItem = ({ test }: ActivityItemProps) => {
+  return (
+    <div className="flex items-start gap-4 p-4 rounded-xl bg-[#0f1419]/50 border border-[rgba(57,255,20,0.1)] hover:border-[rgba(57,255,20,0.3)] transition-all duration-300 group">
+      <div className="mt-1 relative">
+        <div className="w-3 h-3 bg-[#39ff14] rounded-full shadow-[0_0_10px_rgba(57,255,20,0.5)]"></div>
```

Comment:
again do not use direct css tokens instead use css variables only to ensure consistency across platform for all the theme variants